docker/wiremock/stubs/mappings/sdsGetStructuredRecord.json

```diff
@@ -12,9 +12,6 @@
 			}
 		},
 		"headers": {
-			"X-Correlation-Id": {
-				"matches": "[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}"
-			},
```

Comment:
why has this been removed?

Reply:
Confirmed with @bartek-sarul that is should not be required.